csfieldguide/static/interactives/viola-jones-face-detector/js/viola-jones.js

```diff
@@ -317,7 +358,6 @@ function drawGrayscaleImage(img) {
  * Runs the algorithm to find faces and draws a rectange around them.
  */
 function findFaces() {
```

Comment:
Have you confirmed this works for other images? It doesn't find faces on the images I've tried with faces

Reply:
Had a conversation with Jack, it implements a basic version of the algorithm so isn't perfect. Decided we will add some text saying "After you have found the 5 haar like features use the 'find faces' button below to run a basic version of the Viola-Jones algorithm.